rotkehlchen/chain/ethereum/modules/uniswap/v3/decoder.py

```diff
@@ -0,0 +1,102 @@
+from typing import Callable, List, Optional
+
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.assets.asset import EvmToken
+from rotkehlchen.chain.ethereum.decoding.interfaces import DecoderInterface
+from rotkehlchen.chain.ethereum.decoding.structures import ActionItem
+from rotkehlchen.chain.ethereum.decoding.utils import maybe_reshuffle_events
+from rotkehlchen.chain.ethereum.structures import EthereumTxReceiptLog
+from rotkehlchen.chain.ethereum.utils import asset_normalized_value
+from rotkehlchen.constants.assets import A_ETH
+from rotkehlchen.constants.misc import ZERO
+from rotkehlchen.types import EthereumTransaction
+from rotkehlchen.utils.misc import hex_or_bytes_to_address, hex_or_bytes_to_int
+
+from ..constants import CPT_UNISWAP_V3
+
+# https://www.4byte.directory/api/v1/event-signatures/?hex_signature=0xc42079f94a6350d7e6235f29174924f928cc2ac818eb64fed8004e115fbcca67  # noqa: E501
```

Comment:
Can't this be done with `hextring_to_bytes`?

Reply:
yes but I linked this so is not me pasting a ramdom bytes string. Then you can search it and get the method and everything

Reply:
What do you mean? I don't use 4bytes. I just check the logs from etherscan or from my node and just decode the hex to bytes with hexstring_to_bytes.

Reply:
Yes I mean. No problem with that... I do that too. Just that I wanted to leave a reference and left that one. Could link the docs also if you find it better